src/app/views/previous-routes/previous-routes.component.css

```diff
@@ -38,6 +38,13 @@
   border: 0.0625rem dashed black;
 }
 
+.previous-routes__header-fav {
+  text-align: center;
+  flex-basis: 4rem;
+  flex-shrink: 0;
+  margin-left: auto;
```

Comment:
![Screenshot from 2021-02-05 19-43-25](https://user-images.githubusercontent.com/16033930/107075477-7356ab00-67ea-11eb-8c9b-f60e45d91bd4.png)
fav button doesnt align well with the other buttons

Reply:
könnte man den Button vielleicht auch visuell kleiner machen, aber mit dem recht großen Padding, sodass man ihn gut klicken kann, er aber von der breite nicht so viel platz einnimmt?

Reply:
Würde den button nicht verkleinern - der platz wird doch dann trotzdem verbraucht und es ändert sich nichts außer, dass user vielleicht denken sie müssen einen kleinen button treffen?

Reply:
ok, passt :)